bolt-api/subsystems/hasura/migrations/default/1669047367727_add_cpu_warning/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
```

Comment:
No down migration script provided. This could cause issues when rolling back this change.